piker/fsp/_volume.py

```diff
@@ -309,7 +309,7 @@ async def flow_rates(
 
         if period > 1:
             trade_rate_wma = _wma(
-                dvlm_shm.array['trade_count'],
+                dvlm_shm.array['trade_count'][-period:],
```

Comment:
Oh right, this was a totally unrelated patch to improve FSP actor perf since the calc prior was recomputing from all history, which once you have a shite ton of data loaded is pretty costly 😂